src/tools/prte/prte.c

```diff
@@ -885,6 +885,14 @@ int main(int argc, char *argv[])
         PMIX_INFO_LIST_ADD(ret, jinfo, PMIX_DEBUG_STOP_ON_EXEC, NULL, PMIX_BOOL);
     }
 
+    if (pmix_cmd_line_is_taken(&results, PRTE_CLI_STOP_IN_INIT)) {
```

Comment:
This one needs to look at the rank and see if we should be telling it to stop. Probably needs to move down to where we load the per-proc data if we are really going to support per-proc "stop" directives.